Maui.DataGrid/DataGrid.xaml.cs

```diff
@@ -289,6 +289,12 @@ private void SortAndPaginate(SortData? sortData = null)
                     }
                 }
             });
+    
+    /// <summary>
+    /// Gets or sets the Row Tapped Command.
+    /// </summary>
+    public static readonly BindableProperty RowTappedCommandProperty =
+        BindableProperty.Create(nameof(RowTappedCommand), typeof(ICommand), typeof(DataGrid), null);
```

Comment:
You should use `BindablePropertyExtensions.Create<DataGrid, ICommand> ...` for consistency.